src/assets/stylesheets/tables.scss

```diff
@@ -41,6 +41,18 @@ mat-row {
             border-color: #3f51b5 !important;
         }
     }
+
+    &-anchor-cell {
```

Comment:
why are we using `&` in name of class?

Reply:
It is a way to name classes. Refer https://css-tricks.com/the-sass-ampersand/#article-header-id-12